tests/api/test_controller_opdsfeed.py

```diff
@@ -745,7 +740,7 @@ def test_qa_feed(self, circulation_fixture: CirculationControllerFixture):
 
         # For the most part, we're verifying that the expected values
         # are passed in to _qa_feed.
-        assert AcquisitionFeed.groups == kwargs.pop("feed_method")  # type: ignore
+        assert AcquisitionFeed.groups == kwargs.pop("feed_factory")  # type: ignore
```

Comment:
What caused this kwarg to change?

Reply:
I actually don't know. Presumably some recent refactoring inside the CM that would have caused this test to fail had it actually been running. I inspected the `kwargs` array in the debugger, and the bound method was returned via `feed_factory` instead of `feed_method`.